pandas/core/nanops.py

```diff
@@ -200,12 +200,34 @@ def _get_fill_value(dtype, fill_value=None, fill_value_typ=None):
                 return tslibs.iNaT
 
 
-def _get_values(values, skipna, fill_value=None, fill_value_typ=None,
-                isfinite=False, copy=True, mask=None):
+def _maybe_get_mask(values, skipna, mask):
+    """ This function will return a mask iff it is necessary. Otherwise, return
+    None when a mask is not needed.
+    """
+    if (hasattr(values, 'dtype') and is_bool_dtype(values.dtype)):
```

Comment:
is this first test needed?

Reply:
The first test is needed due to a `nanall(Panel)` test case

Reply:
Panel tests should be gone now